test/test_ecc.jl

```diff
@@ -8,7 +8,7 @@ codes = [
     Steane7(),
     Shor9(),
     Perfect5(),
-    Cleve8(),
+    Cleve8(), Gottesman(3), Gottesman(5),
```

Comment:
```suggestion
    Cleve8(),
    Gottesman(3),
    Gottesman(5),
```

Whenever possible, follow the formatting conventions of the code around you